python/adelphi/README.md

```diff
@@ -0,0 +1,107 @@
+# Adelphi tooling
+
+## Commands
+
+### export-cql
+This command extracts schemas for the specified keyspaces from a Cassandra instance and then displays the CQL commands necesary to generate them to standard out.  You can optionally specify an output directory, in which case the CQL commands are written to files within that directory, one file for each keyspace.
+
+The following will display the schemas for the keyspaces "foo" and "bar" on standard out: 
+
+    adelphi --keyspaces foo,bar export-cql
+
+If you wish to store the schemas in a directory "baz" you could use the following instead:
+
+    adelphi --keyspaces foo,bar --output-dir=baz export-cql
+
+### export-gemini
+This command is similar to the "export-cql" command.  The difference is that retrieved schemas are displayed in a format suitable for use with Scylla's [Gemini](https://github.com/scylladb/gemini) tool.
+
+To display Gemini-formatted schemas for the keyspaces "foo" and "bar" use the following:
+
+    adelphi --keyspaces foo,bar export-gemini
+
+And to store these schemas in a directory "baz":
+
+    adelphi --keyspaces foo,bar --output-dir=baz export-gemini
+
+### contribute
+This command automates the workflow of contributing one or more schemas to the Adelphi project.  The [Adelphi schema repository](https://github.com/datastax/adelphi-schemas) is implemented as a Github repository and contributions to this repository take the form of pull requests.  The workflow implemented by this command includes the following steps:
```

Comment:
maybe we can call out here that the `schemas will be *publicly* available in anonymized form` (or something along those lines), such that no one has any surprises

Reply:
This concept made it's way into the "Introduction"; it seemed important enough to warrant an explanation right up front.  Please take a look and let me know if that addresses the concern here.